lib/components/find-a-domain-component.js

```diff
@@ -10,64 +10,80 @@ export default class FindADomainComponent extends BaseContainer {
 		this.declineGoogleAppsLinkSelector = By.className( 'google-apps-dialog__cancel-link' );
 	}
+
 	waitForResults() {
 		const driver = this.driver;
 		const resultsLoadingSelector = By.css( '.domain-suggestion.is-placeholder' );
 		driver.wait( function() {
 			return driverHelper.isElementPresent( driver, resultsLoadingSelector ).then( function( present ) {
-				return ! present;
+				return !present;
 			} );
 		}, this.explicitWaitMS * 2, 'The domain results loading element was still present when it should have disappeared by now.' );
 		return this.checkForUnknownABTestKeys();
 	}
+
 	waitForGoogleApps() {
 		return this.driver.wait( until.elementLocated( this.declineGoogleAppsLinkSelector ), this.explicitWaitMS, 'Could not locate the link to decline google apps.' );
 	}
+
 	searchForBlogNameAndWaitForResults( blogName ) {
 		const searchInputSelector = By.className( 'search__input' );
 		driverHelper.setWhenSettable( this.driver, searchInputSelector, blogName );
 		return this.waitForResults();
 	}
+
 	checkAndRetryForFreeBlogAddresses( expectedBlogAddresses, blogName ) {
 		const self = this;
 		self.freeBlogAddress().then( ( actualAddress ) => {
-			if ( expectedBlogAddresses.indexOf( actualAddress ) < 0 ) {
+			if (expectedBlogAddresses.indexOf( actualAddress ) < 0) {
 				let message = `The displayed free blog address: '${actualAddress}' was not in the expected addresses: '${expectedBlogAddresses}'. Re-searching for '${blogName}' now.`;
 				slackNotifier.warn( message );
 				self.searchForBlogNameAndWaitForResults( blogName );
 			}
 		} );
 	}
+
 	freeBlogAddress() {
 		const freeBlogAddressSelector = By.css( '.domain-suggestion__content h3' );
 		return this.driver.findElement( freeBlogAddressSelector ).getText().then( function( text ) {
 			return text;
 		} );
 	}
+
 	selectDotComAddress( dotComAddress ) {
 		const selector = By.css( `[data-e2e-domain="${ dotComAddress }"]` );
 		this.driver.wait( until.elementLocated( selector ), this.explicitWaitMS, 'Could not locate the select button for the paid address: ' + dotComAddress );
 		const element = this.driver.findElement( selector );
 		this.driver.wait( until.elementIsEnabled( element ), this.explicitWaitMS, 'The paid address button for ' + dotComAddress + ' does not appear to be enabled to click' );
 		return driverHelper.clickWhenClickable( this.driver, selector, this.explicitWaitMS );
 	}
+
 	selectFreeAddress() {
 		const freeAddressSelector = By.css( '.domain-suggestion.is-clickable' );
 		return driverHelper.clickWhenClickable( this.driver, freeAddressSelector, this.explicitWaitMS );
 	}
+
 	selectMapOwnDomain() {
 		const mapOwnSelector = By.css( 'div.domain-mapping-suggestion button' );
 		return driverHelper.clickWhenClickable( this.driver, mapOwnSelector, this.explicitWaitMS );
-	};
+	}
+
+	selectUseOwnDomain() {
+		const useOwnDomain = By.css( '.domain-suggestion.card.is-compact.is-clickable.is-visible.domain-transfer-suggestion' );
```

Comment:
This works, but could be just `.domain-suggestion.card.domain-transfer-suggestion` which is less likely to change (I think) and still unique on the page:

<img width="829" alt="screen shot 2018-01-25 at 4 08 36 pm" src="https://user-images.githubusercontent.com/128826/35373423-0c3c2e9a-01ea-11e8-8d48-ab7ccde4c264.png">

Reply:
Thanks! It also looks more elegant. Fixed.